browser.py

```diff
@@ -243,9 +243,23 @@ def __page_removed_cb(self, notebook, child, pagenum):
 
     def __new_tab_cb(self, browser, url):
         new_browser = self.add_tab(next_to_current=True)
+        self.emit('focus-url-entry')
         new_browser.load_uri(url)
         new_browser.grab_focus()
 
+    def __new_tab_background_cb(self, browser, url):
+        browser = Browser(self._activity)
+        tab_page = TabPage(browser)
+        label = TabLabel(browser)
+        label.connect('tab-close', self.__tab_close_cb, tab_page)
+
+        next_index = self.get_current_page() + 1
+        self.insert_page(tab_page, label, next_index)
+        tab_page.show()
+        self.set_current_page(next_index - 1)
```

Comment:
Some of this code is similar to `add_tab` and what it calls.  The common code could be in a new method called by `add_tab` and `__new_tab_background_cb`.

Also, some of the code in `add_tab` is not included, such as connecting the signals.